### ISSUE TITLE
Expo cli warning about expo package version

### ISSUE THREAD

**Issue by brunon80** [expo/expo-cli#3468]
Hi, today I updated my expo cli to version 4.4.4 and when I ran expo start this warning appeared:
**Your project is in SDK version >= 33.0.0, but the expo package version seems to be older.**

I deleted node_modules and cleared the cache, but the problem persists, however everything seems to be working normally.
I tested on Linux and Mac.

I wonder if anyone else is going through this and if I can do something to solve this possible problem.

**Comment by AKK9** [comment]
I have the same problem after upgrading `expo-cli` from `4.4.3` to `4.4.4` using `yarn global upgrade-interactive` on Linux Ubuntu 20.04.

I also tried `yarn global add expo-cli`.

I also ran `expo update` in my project successfully. (It's a small project only a few days old) The problem still persists.

More info:
```
➜ yarn -v
1.22.5

➜ expo --version
4.4.4

➜ yarn expo --version
4.4.4

➜ yarn info expo | grep version:
version: '41.0.1',
```

**Comment by mannby** [comment]
The problem is actually that there is a missing dependency in package.json, nullthrows. You can work around the problem by installing it globally until the dependency gets added to expo-cli:

npm i -g nullthrows

**Comment by brunon80** [comment]
solved on 4.4.6
